backend/api/pretix/permissions.py

```diff
@@ -0,0 +1,24 @@
+from strawberry.permission import BasePermission
+
+import pretix
+from conferences.models.conference import Conference
+
+
+class IsTicketOwner(BasePermission):
+    message = "You are not allowed to update this ticket."
+
+    def has_permission(self, source, info, **kwargs):
+        pastaporto = info.context.request.pastaporto
+
+        if not pastaporto.is_authenticated:
+            return False
+
+        conference = Conference.objects.get(code=kwargs["conferenceCode"])
```

Comment:
why is the conference code not in snake case here 🤔

Reply:
I didn't investigate, seems strawberry is passing the bare `input` dictionary to the `has_permision`...